iChef-WebUi/src/app/components/recipe-page/recipe-page.module.ts

```diff
@@ -0,0 +1,50 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule, Routes } from '@angular/router';
+
+import {MatGridListModule} from '@angular/material/grid-list';
+import { MatTableModule } from '@angular/material/table';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatButtonModule } from '@angular/material/button';
+import { MatInputModule } from '@angular/material/input';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatIconModule } from '@angular/material/icon';
+import { MatSelectModule } from '@angular/material/select';
+import {MatCardModule} from '@angular/material/card';
+import { MatSortModule } from '@angular/material/sort';
+import { RecipePageComponent } from './recipe-page.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: RecipePageComponent
+  }
+];
+
+@NgModule({
+  declarations: [
+    RecipePageComponent
+    
+  ],
+  imports: [
+    CommonModule,
+    RouterModule.forChild(routes),
+    MatTableModule,
+    MatFormFieldModule,
+    MatButtonModule,
+    MatInputModule,
+    ReactiveFormsModule,
+    MatIconModule,
+    MatSelectModule,
+    MatCardModule,
+    MatInputModule,
+    MatFormFieldModule,
+    MatSortModule,
+    MatGridListModule
+    
+  ],
+  exports: [ //ce comp are pagina
```

Comment:
Maybe make this comment more informative? :D

Reply:
Yes and maybe use english :p